custom-words.txt

```diff
@@ -732,6 +770,8 @@ devicetags
 devicetemplates
 deviceupdate
 devops
+devpay
```

Comment:
Should only need the lowercase versions as the spell check is case-insensitive.

Reply:
I copied the file from https://github.com/Azure/azure-rest-api-specs/blob/main/custom-words.txt

Reply:
I think this is just coming from the changes in main so you can ignore this. I need to de-dupe this entire file anyway.